Monika After Story/game/script-topics.rpy

```diff
@@ -4782,3 +4782,42 @@ label monika_asks_family:
                 m 1f "You can tell me about your family when you're ready, [player]." 
                 m 1j "I love you very much!"                              
     return
+    
```

Comment:
You're missing the `init 5 python` blocks that actually add your topic to the game. Refer to [here](https://github.com/Backdash/MonikaModDev#adding-content)